k8s.gcr.io/images/k8s-staging-provider-azure/OWNERS

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+# Should always mirror the maintainers/reviewers in https://sigs.k8s.io/cluster-api-provider-azure/OWNERS_ALIASES
```

Comment:
```suggestion
# Should always mirror the maintainers/reviewers in https://sigs.k8s.io/cloud-provider-azure/OWNERS
```

Reply:
The code suggestion here will lead to a dead link, but that will be live once we migrate the repo to `kubernetes-sigs`.